lib/reactive_table.js

```diff
@@ -217,7 +217,8 @@ var setup = function () {
 
     context.showNavigation = this.data.showNavigation || this.data.settings.showNavigation || 'always';
     context.showNavigationRowsPerPage = getDefaultTrueSetting('showNavigationRowsPerPage', this.data);
-    
+    context.showRowCount = this.data.showRowCount || this.data.settings.showRowCount || true;
```

Comment:
I think the default should be false for backwards-compatibility, and you can probably use getDefaultFalseSetting